src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -63,29 +63,26 @@ public async Task<Response<ConfigurationSetting>> AddAsync(ConfigurationSetting
 
             Uri uri = BuildUriForKvRoute(setting);
 
-            using (HttpMessage message = _pipeline.CreateMessage(cancellation)) {
-                ReadOnlyMemory<byte> content = Serialize(setting);
-
-                message.SetRequestLine(HttpVerb.Put, uri);
+            var request = _pipeline.CreateRequest();
```

Comment:
With `IDisposable` back on the `HttpRequest`, you may want to consider reintroducing the `using` block.

Reply:
You are right. Missed a couple.